sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/serverless_compute/all_types/automl/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: Serverless in Pipeline - AutoML
+settings:
+  default_compute: azureml:serverless
```

Comment:
looks like we only do test on serverless compute in pipeline settings?

Reply:
Currently yes, will add some node-level test on this in next PR.